src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -169,24 +169,24 @@ private function registerCommonConfiguration(ContainerBuilder $container, array
         $container->setParameter('api_platform.collection.exists_parameter_name', $config['collection']['exists_parameter_name']);
         $container->setParameter('api_platform.collection.order', $config['collection']['order']);
```

Comment:
This needs to be changed, no?

Reply:
Indeed, not sure yet if we should deprecate these parameters?